src/components/containers/EditStudentContainer.js

```diff
@@ -62,7 +66,7 @@ class EditStudentContainer extends Component {
       imageUrl: null,
       campusId: null,
       redirect: true,
-      redirectId: EditStudent.id
+      redirectId: this.props.match.params.id
```

Comment:
this is to get the current student id prev doesn't get the id fast enough cause its in the same func